docs/Developers/Writing_Linter_Bears.rst

```diff
@@ -215,6 +215,13 @@ understands it. This is possible via the ``severity_map`` parameter of
 ``coalib.results.RESULT_SEVERITY`` contains three different values, ``Info``,
 ``Warning`` and ``Error`` you can use.
 
+.. note::
+
+    ``severity_map`` is case insensitive. It means that the
```

Comment:
The line after the period "It means..." seems to be redudant.First line and the example are enough.